sql-plugin/src/main/java/com/nvidia/spark/rapids/format/BufferMeta.java

```diff
@@ -1,3 +1,17 @@
+// Copyright (c) 2024, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 // automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
I excluded all files in the `format` package in the script used to identify files with a missing copyright header in #11723.

Note, these files are generated by a manual `flatc` step. The source files do have a header 
https://github.com/NVIDIA/spark-rapids/blob/351f382c6b5e3cfec6a26edb46381193a1240ae0/sql-plugin/src/main/format/ShuffleCommon.fbs#L1-L13

Reply:
reset the changes of `format/*` files, and added an exclude pattern `sql-plugin/src/main/java/com/nvidia/spark/rapids/format/*` to workflow arguments.